docs/developing.md

```diff
@@ -34,45 +34,28 @@ the service, you can host your own with a series of amazing tools like:
 
 Follow the [install instructions](./install.md) to check out how to install and get this add-on up and running.
 
-## Build scripts
-
-This project uses [Mage](https://magefile.org) as opposed to Make because it's way faster to build and push images, as well as to run tests and other common tasks. Please install [version v1.11.0](https://github.com/magefile/mage/releases/tag/v1.11.0) or above to have access to the task runner.
-
 ### In the Root Directory
 
-The Magefile located in the root directory has targets useful for the whole project. There is another magefile [in the operator directory](../operator/magefile.go), which has targets more specific to the operator module.
-
-The most useful and common commands from the root directory are listed below. Please see the "In the Operator Directory" section for the operator-specific targets. Whther you're in the root or the operator directory, you can always run the following general helper commands:
-
-- `mage -l`: shows a list of all available commands
-- `mage -h <command>`: shows command-specific details
-- `mage -h`: shows the general help
+The Makefile located in the root directory has targets useful for the whole project.
 
-> All commands are case insensitive, so `buildAll` and `buildall` are the same.
+> All commands are case sensitive.
 
-- `mage build`: Builds all the binaries for local testing.
-- `mage test`: Tests the entire codebase
-- `mage dockerBuild`: Builds all docker images
-  - Please see the below "Environment Variables" section for more information on this command
-- `mage dockerPush`: Pushes all docker images, without building them first
-  - Please see the below "Environment Variables" section for more information on this command
-
-### In the Operator Directory
-
-- `mage Manifests`: Builds all the manifest files for Kubernetes, it's important to build after every change
-  to a Kustomize annotation.
-- `mage All`: Generates the operator.
+- `make build`: Builds all the binaries for local testing.
+- `make test`: Tests the entire codebase
```

Comment:
Unit, e2e or both?

Reply:
unit tests, e2e test command is `make e2e-test`, I forgot to add it

Reply:
Let's be more explicit in these docs then
```suggestion
- `make test`: Run all unit tests
```

and document e2e-test

Reply:
done